config/puma.rb

```diff
@@ -1,45 +1,53 @@
 # frozen_string_literal: true
 require 'clowder-common-ruby'
 
-# Puma can serve each request in a thread from an internal thread pool.
-# The `threads` method setting takes two numbers: a minimum and maximum.
-# Any libraries that use thread pools should be configured to match
-# the maximum value specified for Puma. Default is set to 5 threads for minimum
-# and maximum; this matches the default thread size of Active Record.
+min_threads = ENV.fetch('PUMA_MIN_THREADS', 0).to_i
+max_threads = ENV.fetch('PUMA_MAX_THREADS', 16).to_i
+concurrency = ENV.fetch('PUMA_WORKERS', 0).to_i
+
+# Specifies the `host` that puma will listen on to receive requests
 #
-threads_count = ENV.fetch('PUMA_MAX_THREADS') { 5 }
-threads threads_count, threads_count
+set_default_host '0.0.0.0'
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port Settings.web_port
+port ClowderCommonRuby::Config.load.webPort
```

Comment:
Are there any risk when calling the the clowder config load here?
Weren't the clowder config load integrated with `Settings`?

Reply:
When puma loads, clowder config isn't available